features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/assets/api/themes/store/partials/my_applications.hbs

```diff
@@ -3,10 +3,10 @@
         <div class="col-lg-12 white-bg">
             <h3>{{t "Applications"}}</h3>
             <div class="clear"></div>
+            <p>Use applications to subscribe APIs and manage access keys. There is DefaultApplication pre-created to
```

Comment:
is it possible this to take message from i18 library to achieve the localization. May be use t helper will help {{t "your message"}